Bugfix/flow 10355 config page

lmancel opened the pull request:
Will be more easily reviewed commit by commit

Commit:
open config page as a separate window

see FLOW-10355 for more details

Commit:
first serie of wexond fixes

update to electron 17
replaced remote module with @electron/remote
removed f***ing any typing in modified files

TODO Wexond: impossible to navigate after page reload

Commit:
remove publicPath

It's wrong to use this value as resources may be installed anywhere

Azevedo-Dani commented on package.json:
```diff
@@ -127,6 +125,7 @@
     "pretty-bytes": "5.2.0",
     "react": "16.8.6",
     "react-dom": "16.8.6",
+    "react-hook-form": "^7.31.1",
```
"react-hook-form": "7.31.1"

Azevedo-Dani commented on src/config/config.tsx:
```diff
@@ -0,0 +1,172 @@
+        ) : (
+          <></>
+        )}
+        <LabeledInput
```
I don't know if it's because in Polymer it's required but here you can close the Tag directly.
Not have to change just a remark
 <LabeledInput/>

Commit:
fix(wexond) view manager close hook

"clear" was called on window reload, listeners were unregistered
created a "close" function to be called when actually closing the window

Azevedo-Dani commented on src/wexond/main/app-window.ts:
```diff
@@ -57,7 +58,6 @@ export class AppWindow extends KeyboardMixin(BrowserWindow) {
         plugins: true,
         nodeIntegration: true,
         contextIsolation: false,
-        enableRemoteModule: true, // TODO: FLOW-8215
```
That means -> it's done ? 
Could we close this ticket ?
https://jira.taktik.be/browse/FLOW-8215

lmancel replied:
Indeed I can close it

Azevedo-Dani commented on src/wexond/main/view-manager.ts:
```diff
@@ -48,19 +50,14 @@ export class ViewManager {
         const view = this.views[data.tabId]
         if (!view) return
-        let scope: any = view
+        const scope: unknown = getValue(view, data.scope)
```
If I'm not wrong the type is string no ? We can adapt the function getValue with generic return type 
```suggestion
        const scope = getValue<string>(view, data.scope)
```

lmancel replied:
No it's not it may be a function, or anything else for that matter. Thus the `if (!scope || typeof scope !== 'function') return` below

Commit:
updated some dependencies

otherwise install on Windows is impossible............

Commit:
fix(vlc) pass log level to vlc command

program was crashing because args array was not as long as expected

Commit:
misc(config) react element auto closing tags

Commit:
fix(keyboard) repair ipc communication

Also set AZERTY layout (not configurable yet)

Commit:
misc(build) slight conf changes

Commit:
slight rework on debug mode info -> provide it to wexond

Windows -> set pseudo transparent background to prevent click-through

Commit:
fix(wexond) safeguards for extensions and filters

ensure that any error in their init does not block everything

Commit:
package-lock

Commit:
1.3.0-beta2

Commit:
update IFlowrDesktopConfigUserPreferences

some values were missing
